index.html

```diff
@@ -61,13 +69,116 @@ <h2 class="section-title">What's Next</h2>
         <ul class="skills-list">
           <li>Object Oriented Programming</li>
           <li>React JS</li>
+          <li>Native JS Web Components</li>
           <li>Flutter</li>
           <li>Node JS</li>
           <li>Express JS</li>
           <li>MongoDB</li>
           <li>Firebase</li>
         </ul>
       </section>
+      <section class="projects">
+        <h2 class="section-title">Projects</h2>
+        <div class="projects-container">
+
+          <div class="project-wrapper project-1">
+            <h4 class="project-title">To Do App</h4>
+            <p class="project-description"><span class="lead-text">Features:<br /></span>HTML5 (Semantic, Accessible) - CSS3 (Mobile
+              First
+              Responsive Layout, Flexbox, CSS
+              Grid) - JS
+              (DOM Manipulation)
+            </p>
+            <div class="btn-wrapper">
+              <a href="https://rotunstall.dev/to-do-app" target="_blank" class="live-site">Live Site</a>
+              <a href="https://github.com/rotunstall/To-Do-List" target="_blank" class="site-code">GitHub
+                Repo</a>
+            </div>
+
+          </div>
+          <div class="project-wrapper project-2">
+            <h4 class="project-title">NBA Team Profile</h4>
+            <p class="project-description"><span class="lead-text">Features:<br /></span>HTML5 (Semantic, Accessible) - CSS3 (Mobile
+              First
+              Responsive Layout, Flexbox, CSS
+              Grid) - JS
+              (DOM Manipulation,
+              AJAX, Promises)
+            </p>
+            <div class="btn-wrapper">
+              <a href="https://rotunstall.dev/nba-teams-profile-app" target="_blank" class="live-site">Live Site</a>
+              <a href="https://github.com/rotunstall/NBA-League-Pass" target="_blank" class="site-code">GitHub
+                Repo</a>
+            </div>
+          </div>
+          <!-- <div class="project-wrapper project-3">
```

Comment:
I would usually advise against leaving commented code or markup, but in this context it's not too big of a deal.

Will just leave this comment here :)